src/components/diagnosticReportView/RegionalMfdView.tsx

```diff
@@ -50,33 +56,72 @@ const RegionalMfdView: React.FC<RegionalMfdViewProps> = ({
     setRegionalViewSelections(filtered);
   }, [regionalViews]);
 
+  useEffect(() => {
+    const filtered = solutionMfdOptions.filter((option) => {
+      return regionalViews.includes(option.displayName);
+    });
+    setNonRegionalViewSelections(filtered);
+  }, [regionalViews]);
+
   const getRegionalMfdUrl = (id: string, finalPath: string): string => {
     return buildUrl(process.env.REACT_APP_REPORTS_URL, {
       path: `/opensha/DATA/${id}/solution_report/resources/${finalPath}`,
     });
   };
 
   const regionalSolutionMfdDisplayNames: string[] = [];
+  const solutionMfdDisplayNames: string[] = [];
 
   regionalSolutionMfdOptions.map((option) => {
     regionalSolutionMfdDisplayNames.push(option.displayName);
   });
 
-  return (
-    <Root>
-      <MultiSelect
-        name="Regional Solution MFD"
-        selected={regionalViews}
-        options={regionalSolutionMfdDisplayNames}
-        setOptions={setRegionalViews}
-      />
-      <div className={classes.imageContainer}>
-        {regionalViewSelections.map((option) => (
-          <img key={option.path} className={classes.image} src={getRegionalMfdUrl(id, option.path)} alt={option.path} />
-        ))}
-      </div>
-    </Root>
-  );
+  solutionMfdOptions.map((option) => {
+    solutionMfdDisplayNames.push(option.displayName);
+  });
+
+  if (regional) {
+    return (
+      <Root>
+        <MultiSelect
+          name="Regional Solution MFD"
+          selected={regionalViews}
+          options={regionalSolutionMfdDisplayNames}
+          setOptions={setRegionalViews}
+        />
+        <div className={classes.imageContainer}>
+          {regionalViewSelections.map((option) => (
+            <img
+              key={option.path}
+              className={classes.image}
+              src={getRegionalMfdUrl(id, option.path)}
+              alt={option.path}
+            />
+          ))}
+        </div>
+      </Root>
+    );
+  } else
+    return (
+      <Root>
+        <MultiSelect
+          name="Solution MFD"
+          selected={regionalViews}
```

Comment:
Should we use a different state for this selection?

Reply:
have done this was quite big change but works, fixes multiple selected bug aswell